### PR TITLE
Feature/update svdb

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR adds | fixes:

- See above

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
Install: ✅ 
```
[[A[0|0|584] [henrik.stranneheim@hasta:~/git/MIP] [base] feature/update_svdb 2s ± perl mip install --envn D_mip_hs --sp svdb
[INFO] 2021/03/26 14:53:06 MIP_INSTALL - MIP Version: v9.1.3
[INFO] 2021/03/26 14:53:06 MIP_INSTALL - Writing log messages to: /home/henrik.stranneheim/git/MIP/mip_log/2021-03-26/mip_install_2021-03-26T14:53:06.log
[INFO] 2021/03/26 14:53:06 MIP_INSTALL - Installing pipelines: rd_dna rd_rna
[INFO] 2021/03/26 14:53:06 MIP_INSTALL - Installing MIP into environment: D_mip_hs
[INFO] 2021/03/26 14:53:06 MIP_INSTALL - Caching image: svdb
[INFO] 2021/03/26 14:53:08 MIP_INSTALL - Testing programs in image: svdb
[INFO] 2021/03/26 14:53:12 MIP_INSTALL - 	Successfully launched: svdb 2.4.0,
[INFO] 2021/03/26 14:53:12 MIP_INSTALL - Finished installing MIP
```

<details>
<Summary>
RUN ✅ 
</summary>
<pre>
[INFO] 2021/03/26 15:01:02 MIP_ANALYSE - [SV COMBINEVARIANTCALLSETS]
[INFO] 2021/03/26 15:01:02 MIP_ANALYSE - Creating sbatch script for sv_combinevariantcallsets and writing script file(s) to: /home/henrik.stranneheim/develop/justhusky-selected-regions/analysis/scripts/justhusky-selected-regions/sv_combinevariantcallsets/sv_combinevariantcallsets_justhusky-selected-regions.3.sh
[INFO] 2021/03/26 15:01:02 MIP_ANALYSE - Sbatch script sv_combinevariantcallsets data files will be written to: /home/henrik.stranneheim/develop/justhusky-selected-regions/analysis/justhusky-selected-regions/sv_combinevariantcallsets
[INFO] 2021/03/26 15:01:02 MIP_ANALYSE - Sbatch script submitted, job id: 1049453
[INFO] 2021/03/26 15:01:02 MIP_ANALYSE - To check status of job, please run 'squeue -j 1049453'
[INFO] 2021/03/26 15:01:02 MIP_ANALYSE - To cancel job, please run 'scancel 1049453'
[1|11|592] [henrik.stranneheim@hasta:~] [base] 130 $ ejobinfo 1049453 | cat
Lines   total/split/realigned/skipped:	170/0/0/0
Lines   total/split/realigned/skipped:	154/0/0/0
/bin/sh: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
Lines   total/split/realigned/skipped:	1811/0/0/0
[1|11|592] [henrik.stranneheim@hasta:~] [base] $ scontrol show jobid=1049453 -d
JobId=1049453 JobName=sv_combinevariantcallsets_justhusky-selected-regions
   UserId=henrik.stranneheim(1265) GroupId=users(100) MCS_label=N/A
   Priority=124 Nice=0 Account=development QOS=low WCKey=*default
   JobState=COMPLETED Reason=None Dependency=(null)
   Requeue=1 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
   DerivedExitCode=0:0
   RunTime=00:00:33 TimeLimit=02:00:00 TimeMin=N/A
   SubmitTime=2021-03-26T15:01:02 EligibleTime=2021-03-26T15:01:02
   AccrueTime=2021-03-26T15:01:02
   StartTime=2021-03-26T15:01:02 EndTime=2021-03-26T15:01:35 Deadline=N/A
   PreemptTime=None SuspendTime=None SecsPreSuspend=0
   LastSchedEval=2021-03-26T15:01:02
   Partition=core AllocNode:Sid=hasta:50851
   ReqNodeList=(null) ExcNodeList=(null)
   NodeList=gpu-compute-0-0
   BatchHost=gpu-compute-0-0
   NumNodes=1 NumCPUs=1 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   TRES=cpu=1,mem=5G,node=1,billing=1
   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
     Nodes=gpu-compute-0-0 CPU_IDs=17 Mem=5120 GRES_IDX=
   MinCPUsNode=1 MinMemoryNode=5G MinTmpDiskNode=0
   Features=(null) DelayBoot=00:00:00
   OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
   Command=/home/henrik.stranneheim/develop/justhusky-selected-regions/analysis/scripts/justhusky-selected-regions/sv_combinevariantcallsets/sv_combinevariantcallsets_justhusky-selected-regions.3.sh
   WorkDir=/home/henrik.stranneheim
   StdErr=/home/henrik.stranneheim/develop/justhusky-selected-regions/analysis/justhusky-selected-regions/sv_combinevariantcallsets/info/sv_combinevariantcallsets_justhusky-selected-regions.3.stderr.txt
   StdIn=/dev/null
   StdOut=/home/henrik.stranneheim/develop/justhusky-selected-regions/analysis/justhusky-selected-regions/sv_combinevariantcallsets/info/sv_combinevariantcallsets_justhusky-selected-regions.3.stdout.txt
   Power=

File: documentation/Setup.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ You can speed up, for instance, the Readonly module by also installing the compa
 - [Samtools] (version: 1.11)
 - [SMNCopyNumberCaller] (version: v1.1.1)
 - [Stranger] (version: 0.7)
-- [Svdb] (version: 2.2.0)
+- [Svdb] (version: 2.2.4)
```
Comment (jemten) [inline documentation/Setup.md:64]:
```suggestion
- [Svdb] (version: 2.4.0)
```

**Review by jemten — APPROVED** [review]
👍

File: documentation/Setup.md (hunk 1 above)
Comment (henrikstranneheim) [reply documentation/Setup.md:64]: Good catch

**Pull request merged**